README.md

```diff
@@ -52,9 +52,7 @@ Every PR must be prepared using the following commands:
 ```bash
 cd $GOPATH/src/github.com/go-clang/gen
 scripts/switch-clang-version.sh 3.4
-make install
-make test
-make lint
+make pr
```

Comment:
fine by me. any suggestion?
`make test` is already taken.
`make check` is perhaps too vague (and perhaps easily confused with `test`)